Bottom: 60c5e3d8bf0af74fe3b1fde4c1efb56663e3cd07
Top:    aef744839e76ad6b5185c34cebd13871736d90ee
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-20 23:41:02 +0000

Refresh of more-work-in-progress

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 7cd48fb..83d0b98 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -3076,8 +3076,6 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
-static void clear_marks (POLYAREA * p);
-
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
